src/Crank/CoreWCF.Perf.yml

```diff
@@ -0,0 +1,189 @@
+jobs:
+  CoreWCFPerfClient:
+    source:
+      repository: https://github.com/CoreWCF/CoreWCF.git
+      branchOrCommit: master
+      project:  Src/Crank/CoreWCFPerfClient/CoreWCFPerfClient.csproj
+    isConsoleApp: true
+    waitForExit: true
+    variables:
+      Duration: 300
+      requests: 3000
+    arguments: "binding:{{Binding}} serviceurl:{{Protocol}}://{{serverAddress}}:{{serverPort}}/SayHello/SayHello.svc perfmeasurementduration:{{Duration}} Requests:{{requests}} closeandopen:{{closeandopen}}"
+  CoreWCFPerfService:
+    source:
+      repository: https://github.com/CoreWCF/CoreWCF.git
+      branchOrCommit: master
+      project: Src/Crank/CoreWCFPerfService/CoreWCFPerfService.csproj
+    arguments: "baseaddress:{{Protocol}}://{{serverAddress}}:{{serverPort}}"
+    isConsoleApp: true
+
+scenarios:
+  BasicHttpWithCloseAndOpen:  
+    application:
+      job: CoreWCFPerfService
+      variables:
+        Protocol: http
+    load:
+      job: CoreWCFPerfClient
+      variables:
+        Protocol: http
+        Binding: BasicHttp
+        closeandopen: true
+
+  NetTcpWithCloseAndOpen: 
+    application:
+      job: CoreWCFPerfService
+      variables:
+        Protocol: net.tcp
+    load:
+      job: CoreWCFPerfClient
+      variables:
+        Protocol: net.tcp
+        Binding: NetTcp
+        closeandopen: true
+
+  BasicHttpWithoutCloseAndOpen:   
+    application:
+      job: CoreWCFPerfService
+      variables:
+        Protocol: http
+    load:
+      job: CoreWCFPerfClient
+      variables:
+        Protocol: http
+        Binding: BasicHttp
+        closeandopen: false
+
+  NetTcpWithoutCloseAndOpen:    
+    application:
+      job: CoreWCFPerfService
+      variables:
+        Protocol: net.tcp
+    load:
+      job: CoreWCFPerfClient
+      variables:
+        Protocol: net.tcp
+        Binding: NetTcp
+        closeandopen: false
+
+profiles:
```

Comment:
We can remove this part and import existing profile configuration: https://github.com/aspnet/Benchmarks/blob/main/scenarios/aspnet.profiles.yml

Reply:
> We can remove this part and import existing profile configuration: https://github.com/aspnet/Benchmarks/blob/main/scenarios/aspnet.profiles.yml

The changes updated in this PR: https://github.com/CoreWCF/CoreWCF/pull/796, please help review.

Reply:
> We can remove this part and import existing profile configuration: https://github.com/aspnet/Benchmarks/blob/main/scenarios/aspnet.profiles.yml

duplicate key "aspnet-citrine-win-relay" exist in this profiles